### PR TITLE
mailchimp and toCamelCase fixed

### PR DESCRIPTION
**Pull request by alexgbor**

### CODE REVIEW

File: staff/alexgbor/js/challenges/to-camel-case/scripts/toCamelCase.js [hunk 1]
```diff
@@ -2,13 +2,12 @@
```
Comment (agandia9) [inline staff/alexgbor/js/challenges/to-camel-case/scripts/toCamelCase.js:2]: use strict in all js files!

Comment (alexgbor) [reply staff/alexgbor/js/challenges/to-camel-case/scripts/toCamelCase.js:2]: I'm on it, thanks!

Comment (alexgbor) [reply staff/alexgbor/js/challenges/to-camel-case/scripts/toCamelCase.js:2]: Done

**Review by agandia9 — COMMENTED** [review]
ok!

**Pull request merged**